arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesViewController.swift

```diff
@@ -166,11 +162,6 @@ class CreateSymbolStylesFromWebStylesViewController: UIViewController, UIAdaptiv
         // Ensure that the settings are shown in a popover on small displays.
         return .none
     }
-    
-    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
-        // Release the data source when the table view controller is dismissed.
-        symbolsDataSource = nil
```

Comment:
Don't release the symbols data source as we are using it to hold the legend items.